packages/cozy-harvest-lib/src/helpers/oauth.js

```diff
@@ -160,6 +167,7 @@ const getAppSlug = client => {
  * @param  {string} redirectSlug The app we want to redirect the user on after the end of the flow
  * @param  {Object} extraParams some extra parameters to add to the query string
  * @param  {Boolean} reconnect Are we trying to reconnect an existing account ?
+ * @param  {Boolean} manage Are we trying to manage an existing account ?
```

Comment:
can you add 'for bi account' or something like that please ?

Reply:
I would prefer not to do that regarding https://github.com/cozy/cozy-libs/pull/1852#discussion_r1001466358